aurora/interface/sample/from_id.py

```diff
@@ -14,6 +10,16 @@
 
 
 class SampleFromId(ipw.VBox):
+    """
+    A sample selection widgets container used in the main panel to
+    select samples for an experiment by sample id.
+
+    TODO implement creation and labeling of sample nodes.
+    TODO store submitted samples in a group
+    TODO retrieve a node if it was already created.
+
+    TODO refactor filters to separate class
```

Comment:
I would maybe open issues for these and the other `TODO` comments (you can leave the comments if you want, I just find it a bit extra mental load having to remember to update these here too).